### PR TITLE
improvement in efk-logging.html

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1699177

### CODE REVIEW

File: modules/efk-logging-about-crd.adoc [hunk 1]
```diff
@@ -3,24 +3,24 @@
...
 
 You should never need to modify this CRD. To make changes to your deployment, create and modify a specific Custom Resource (CR).
 Instructions for creating or modifying a CR are provided in this documentation as appropriate.
 
-The following is an example of a typical Custom Resource Definition for cluster logging.
+The following is an example of a typical Custom Resource for cluster logging.
 
 [id="efk-logging-configuring-about-sample-{context}"]
-.Sample ClusterLogging CRD
+.Sample ClusterLogging CR
```
Comment (mburke5678) [inline modules/efk-logging-about-crd.adoc]: @anpingli I changed this to Sample Cluster Logging CR. This is CR, not the CRD, I believe.

**Comment by mburke5678** [comment]
@anpingli PTAL

**Review by anpingli — APPROVED** [review]
LGTM

**Comment by vikram-redhat** [comment]
@mburke5678 - merged into 4.0. 

Nothing we can do now, but please make sure to create a new PR for the 4.1 branch to incorporate this change otherwise it would be lost. I am deleting the 4.0 branch so this doesn't happen again. You should be able to just do a CP of the commit.

**Pull request merged**